src/inmanta/app.py

```diff
@@ -834,11 +839,64 @@ def cmd_parser() -> argparse.ArgumentParser:
         if cmd_options["parser_config"] is not None:
             cmd_options["parser_config"](cmd_subparser, parent_parsers)
         cmd_subparser.set_defaults(func=cmd_options["function"])
+        cmd_subparser.set_defaults(component=cmd_options["component"])
         cmd_subparser.set_defaults(require_project=cmd_options["require_project"])
 
     return parser
 
 
+def default_log_config_parser(parser: ArgumentParser, parent_parsers: abc.Sequence[ArgumentParser]) -> None:
+    subparser = parser.add_subparsers(title="subcommand", dest="cmd")
+
+    subparser.add_parser(
+        "server",
+        help="Output default log file for the server, given the current configuration file and options",
+        parents=parent_parsers,
+    )
+    subparser.add_parser(
+        "scheduler",
+        help="Output default log file template for the scheduler, given the current configuration file and options",
+        parents=parent_parsers,
+    )
+    subparser.add_parser(
+        "compiler",
+        help="Output default log file template for the compiler, given the current configuration file and options",
+        parents=parent_parsers,
+    )
+
+
+@command("default_logging_config", help_msg="Print the default log config", parser_config=default_log_config_parser)
```

Comment:
I think `generate_default_loging_config` or some other prefix like `print_`, `export_` or `print_` would convey better what this does.

I would also extend the help message with: `Print the default log config for the provided component`